darktable

Latest darktable still not buildable.
Output:
````
[ 45%] [32m[1mLinking C shared library libdarktable.dll[0m
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<char>::deallocate(char*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `void std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_construct<char const*>(char const*, char const*, std::forward_iterator_tag)':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_string.tcc:219: undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_create(unsigned long long&, unsigned long long)'
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_string.tcc:212: undefined reference to `std::__throw_logic_error(char const*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<wchar_t>::deallocate(wchar_t*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<char>::deallocate(char*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<wchar_t>::deallocate(wchar_t*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<char>::deallocate(char*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj):C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: more undefined references to `operator delete(void*)' follow
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `void std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_construct<char const*>(char const*, char const*, std::forward_iterator_tag)':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_string.tcc:219: undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_create(unsigned long long&, unsigned long long)'
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_string.tcc:219: undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_create(unsigned long long&, unsigned long long)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `void std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> >::_M_construct<wchar_t*>(wchar_t*, wchar_t*, std::forward_iterator_tag)':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_string.tcc:219: undefined reference to `std::__cxx11::basic_string<wchar_t, std::char_traits<wchar_t>, std::allocator<wchar_t> >::_M_create(unsigned long long&, unsigned long long)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::basic_ostream<char, std::char_traits<char> >& std::operator<< <std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*)':
C:/building/msys64/mingw64/include/c++/7.2.0/ostream:561: undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::__ostream_insert<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*, long long)'
C:/building/msys64/mingw64/include/c++/7.2.0/ostream:561: undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::__ostream_insert<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*, long long)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::basic_ostream<char, std::char_traits<char> >& std::endl<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&)':
C:/building/msys64/mingw64/include/c++/7.2.0/ostream:591: undefined reference to `std::ostream::put(char)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::basic_ostream<char, std::char_traits<char> >& std::flush<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&)':
C:/building/msys64/mingw64/include/c++/7.2.0/ostream:613: undefined reference to `std::ostream::flush()'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::basic_ios<char, std::char_traits<char> >::setstate(std::_Ios_Iostate)':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_ios.h:158: undefined reference to `std::basic_ios<char, std::char_traits<char> >::clear(std::_Ios_Iostate)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::ctype<char>::widen(char) const':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/locale_facets.h:876: undefined reference to `std::ctype<char>::_M_widen_init() const'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::ctype<char> const& std::__check_facet<std::ctype<char> >(std::ctype<char> const*)':
C:/building/msys64/mingw64/include/c++/7.2.0/bits/basic_ios.h:50: undefined reference to `std::__throw_bad_cast()'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<char>::deallocate(char*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<wchar_t>::deallocate(wchar_t*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<char>::deallocate(char*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `__gnu_cxx::new_allocator<wchar_t>::deallocate(wchar_t*, unsigned long long)':
C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: undefined reference to `operator delete(void*)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj):C:/building/msys64/mingw64/include/c++/7.2.0/ext/new_allocator.h:125: more undefined references to `operator delete(void*)' follow
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `dt_exif_get_thumbnail':
C:/repo/mingw-w64-darktable/src/darktable-2.4.1/src/common/exif.cc:964: undefined reference to `__cxa_begin_catch'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `std::basic_ostream<char, std::char_traits<char> >& std::operator<< <std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*)':
C:/building/msys64/mingw64/include/c++/7.2.0/ostream:561: undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::__ostream_insert<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*, long long)'
CMakeFiles/lib_darktable.dir/objects.a(exif.cc.obj): In function `dt_exif_get_thumbnail':

.....
````

--- a/mingw-w64-darktable/PKGBUILD
+++ b/mingw-w64-darktable/PKGBUILD
@@ -31,6 +31,7 @@ depends=("${MINGW_PACKAGE_PREFIX}-dbus-glib"
          "${MINGW_PACKAGE_PREFIX}-pugixml"
          "${MINGW_PACKAGE_PREFIX}-sqlite3"
          "${MINGW_PACKAGE_PREFIX}-iso-codes"
+         "${MINGW_PACKAGE_PREFIX}-zlib"
         )
 options=('!strip')
 source=(${_realname}-${pkgver}.tar.xz::"https://github.com/darktable-org/${_realname}/releases/download/release-${pkgver}/${_realname}-${pkgver}.tar.xz")
@@ -57,6 +58,7 @@ build() {
   ${MINGW_PREFIX}/bin/cmake \
     -G'MSYS Makefiles' \
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+    -DBINARY_PACKAGE_BUILD=ON \
     "${extra_config[@]}" \
     ../${_realname}-${pkgver}
 

